frontend-shared/styles/mixins/focus.scss

```diff
@@ -1,3 +1,22 @@
+@use './utils' as utils;
+@use '../variables' as var;
+
+/**
+ * Add stylized focus to interactive elements such <input> or <textarea>
+ */
+@mixin outline {
+  @include utils.transition--fast(box-shadow);
+  &:focus {
+    outline: 0;
+    box-shadow: 0 0 0 2px var.$color-focus-outline inset;
```

Comment:
`inset` is critical because otherwise the box-shadow can get overlapped by adject elements. That is one aspect that native styling has a clear advantage. This also makes its very hard to see the 2px border radius on small elements, but it is there, just less pronounced.